tools/harness-thci/README.md

```diff
@@ -28,3 +29,28 @@ new THCI implementation.
 6. Drag OpenThread: TI CC2538DK reference device to Test Bed list with desired number.
 
 7. Select one or multiple test cases to execute.
+
+
+## OpenThread WpanCtl Environment Setup ##
+
+1. Copy the "OpenThread.png" to \GRL\Thread1.1\Web\images folder.
+
+2. Copy the "deviceInputFields.xml" to \GRL\Thread1.1\Web\data folder.
+
+3. Copy the "OpenThread_WpanCtl.py" to \GRL\Thread1.1\Thread_Harness\THCI folder.
+
+4. Connect the NCP board (nRF52840) to Raspberry Pi's usb port and make sure wpanctl command work.
```

Comment:
"...to the Raspberry Pi's USB port and verify that the `wpanctl` command works."